src/lib/components/LandingPage.svelte

```
@@ -4,9 +4,12 @@
 	import { Checkbox } from '$lib/components/ui/checkbox/index.js';
 	import * as Dialog from '$lib/components/ui/dialog/index.js';
 	import * as Table from '$lib/components/ui/table/index.js';
+	import PreTestTimer from './PreTestTimer.svelte';
 
 	const { testDetails } = $props();
 	let isChecked = $state(false);
+	let showPreTestTimer = $state(false);
+	let candidateData = $state<any>(null);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)